### DISCUSSION TITLE
Apple Wide Color Sharing Profile?

### DISCUSSION THREAD

**Discussion by Jmuccigr** [ImageMagick/ImageMagick#2553] [Help]
I use Hugin.app on a Mac to do some work creating photo-mosaics and the like.

When I use photos from my iPhone, I sometimes get complaints from the app that a photo has no color profile, even though others in the group do. If I run `identity -verbose` on two separate images taken at the same time, only one of which gets a complaint, I find the only difference that seems relevant is this in the accepted image:

```
    icc:copyright: Copyright Apple Inc., 2016
    icc:description: Apple Wide Color Sharing Profile
```

So I'm wondering if it's possible to set this with IM.

PS No idea why Photos.app exports different info for each photo, so will be happy for a way to avoid that.

**Comment by snibgo** [comment]
If you have a file photo1.jpg that has an embedded profile, you can make an ICC from it:
```
magick photo1.jpg myprof.icc
```
If you have photo2.jpg that is encoded with that profile but has no embedded profile, you can assign the profile:
```
magick photo2.jpg -profile myprof.icc photo2.jpg
```
(In that command, I overwrite the input file.)

Beware: if photo2.jpg is actually encoded as sRGB or other colorspace, then assigning the wrong profile will lead to bad colours.

**Comment by Jmuccigr** [comment]
No worries. They should be the same and in fact this worked for me (though enblend, part of Hugin, complained about the profile).
